canonical-web-and-design/snapcraft.io
# Remove custom patterns

Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1097

## Done
- Remove custom `media-object` and `media-object--large`
- Remove custom `social-icons`
- Remove custom `table-key-value`
- Remove custom `inlint-list--compact`
- Remove custom `p-card` originally from insights
- Remove custom `p-tabs` used on publisher page
- Remove custom 404 styling (that wasn't used)
- Remove account-snaps styling
- Move footer icon from `_snapcraft_footer` to `_snapcraft_patterns_icons`
- Remove custom `p-form-help-text`
- Remove custom `p-case-study`
- Remove custom `snap-row`
- Remove custom `p-grid`
- Remove custom `p-code-block`
- Updated relevant templates

## QA
- Pull the branch
- Make sure the scss builds... Next up - going through each page.
